src/main/java/ch/njol/skript/effects/EffKill.java

```diff
@@ -26,44 +26,35 @@
 		"Note: This effect does not set the entity's health to 0 (which causes issues), but damages the entity by 100 times its maximum health."})
 @Examples({"kill the player",
 		"kill all creepers in the player's world",
-		"kill all endermen, witches and bats"})
+		"kill all endermen, witches and bats",
+})
 @Since("1.0")
 public class EffKill extends Effect {
 
 	static {
 		Skript.registerEffect(EffKill.class, "kill %entities%");
 	}
 	
-	// Absolutely make sure it dies
-	public static final int DAMAGE_AMOUNT = Integer.MAX_VALUE;
```

Comment:
Just a general rule in API, you should add this back with a deprecated and scheduled for removal annotation. It's highly unlikely this is used by any consumers of the API, but nevertheless good practice in case it does, so the consumer has more opportunity to change their use case

Reply:
That was legacy code, and I thought I should remove it, but it really didn't matter, since this is like the least important place to apply it.